[2023-10-01 09:58:49,005] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: postgres_v1.create_postgres_connection 2021-08-28T00:00:00+00:00 [queued]>
[2023-10-01 09:58:49,845] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: postgres_v1.create_postgres_connection 2021-08-28T00:00:00+00:00 [queued]>
[2023-10-01 09:58:49,852] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2023-10-01 09:58:49,864] {taskinstance.py:1043} INFO - Starting attempt 1 of 6
[2023-10-01 09:58:49,869] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2023-10-01 09:58:50,358] {taskinstance.py:1063} INFO - Executing <Task(PostgresOperator): create_postgres_connection> on 2021-08-28T00:00:00+00:00
[2023-10-01 09:58:50,895] {standard_task_runner.py:52} INFO - Started process 44110 to run task
[2023-10-01 09:58:51,779] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'postgres_v1', 'create_postgres_connection', '2021-08-28T00:00:00+00:00', '--job-id', '2616', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/1_postgresql.py', '--cfg-path', '/tmp/tmpkj1gvjey', '--error-file', '/tmp/tmpl_zor772']
[2023-10-01 09:58:51,856] {standard_task_runner.py:77} INFO - Job 2616: Subtask create_postgres_connection
[2023-10-01 09:59:17,576] {logging_mixin.py:104} INFO - Running <TaskInstance: postgres_v1.create_postgres_connection 2021-08-28T00:00:00+00:00 [running]> on host d4ed655215d1
[2023-10-01 09:59:25,272] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=svastikkka
AIRFLOW_CTX_DAG_ID=postgres_v1
AIRFLOW_CTX_TASK_ID=create_postgres_connection
AIRFLOW_CTX_EXECUTION_DATE=2021-08-28T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-28T00:00:00+00:00
[2023-10-01 09:59:25,565] {postgres.py:69} INFO - Executing: create table if not exists dags_run(
        dt  date,
        dag_id character varying,
        primary key (dt,dag_id)
        )
[2023-10-01 09:59:26,319] {base.py:74} INFO - Using connection to: id: posgres_testing. Host: host.docker.internal, Port: 5432, Schema: testing, Login: airflow, Password: XXXXXXXX, extra: None
[2023-10-01 09:59:26,701] {dbapi.py:180} INFO - Running statement: create table if not exists dags_run(
        dt  date,
        dag_id character varying,
        primary key (dt,dag_id)
        ), parameters: None
[2023-10-01 09:59:26,835] {dbapi.py:186} INFO - Rows affected: -1
[2023-10-01 09:59:26,858] {postgres.py:73} INFO - NOTICE:  relation "dags_run" already exists, skipping

[2023-10-01 09:59:28,338] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=postgres_v1, task_id=create_postgres_connection, execution_date=20210828T000000, start_date=20231001T095849, end_date=20231001T095928
[2023-10-01 09:59:31,422] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-10-01 09:59:31,993] {local_task_job.py:146} INFO - Task exited with return code 0
